core/src/test/java/org/mule/transport/polling/watermark/WatermarkSelectorTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.transport.polling.watermark;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+import org.mule.transport.polling.watermark.selector.MaxValueWatermarkSelector;
+import org.mule.transport.polling.watermark.selector.MinValueWatermarkSelector;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+
+import org.junit.Test;
+
+public class WatermarkSelectorTest
```

Comment:
Classes must be named with "TestCase"suffix and must extend from AbstractMuleTestCase